grails-app/services/transmart/mydas/VcfInfoService.groovy

```diff
@@ -53,7 +52,7 @@ class VcfInfoService  extends  VcfServiceAbstract {
                 // end pos
                 val.position.toInteger(),
                 // value - this is where we place the value from the info field
-                (val.infoFields[infoField]?:0) as double,
+                infoFieldValue as double,

```

Comment:
Perhaps you could add a comment for a future reader of the code, that it's really ok to try to read the info field content as double without guarding for exceptions mentioning the reasons we discussed